src/as2fm/jani_generator/scxml_helpers/top_level_interpreter.py

```diff
@@ -42,13 +42,23 @@
 
 @dataclass()
 class FullModel:
+    # The maximum time the model is allowed to run, in nanoseconds
     max_time: Optional[int] = None
+    # Max size of "dynamic" arrays defined in the SCXML models
     max_array_size: int = field(default=100)
+    # Tick rate for the loaded BT
     bt_tick_rate: float = field(default=1.0)
+    # Whether to keep ticking the BT after it returns SUCCESS / FAILURE
+    bt_tick_not_running: bool = field(default=False)
```

Comment:
```suggestion
    bt_tick_when_not_running: bool = field(default=False)
```
Just a suggestion ...

Reply:
Done